src/components/atoms/Icon/Icon.tsx

```diff
@@ -17,7 +18,7 @@ const {
   DelayIcon,
 } = Icons;
 
-const iconsMap: any = {
+const iconsMap: {[key in string]: React.FC<{}>} = {
```

Comment:
if we don't know the props of the FC, lets not use the generic. e.g.  {[key in string]: React.FC} = {

Reply:
or use JSX.Element instead?!